src/syncerd/bitcoin_syncer.rs

```diff
@@ -1008,11 +1065,13 @@ impl Synclet for BitcoinSyncer {
             let electrum_server = electrum_server.clone();
             std::thread::spawn(move || {
                 use tokio::runtime::Builder;
+                info!("building tokio syncer runtime");
                 let rt = Builder::new_multi_thread()
                     .worker_threads(2)
                     .enable_all()
                     .build()
-                    .unwrap();
+                    .expect("failed to build tokio runtime");
+                info!("completed tokio syncer runtime");
```

Comment:
make these trace

Reply:
Done in [c48ab96](https://github.com/farcaster-project/farcaster-node/pull/614/commits/c48ab96aaafa4989a93cb51b3c5e82009c3afddb)